### PR TITLE
Fix/dst

### PR DESCRIPTION
**Pull request by aleontiev**

### CODE REVIEW

File: schedule/requirements.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ argparse==1.1
 django-coverage==1.2.4
 django-nose==1.1
 nose==1.3.0
-python-dateutil==2.1
+altschool-python-dateutil==2.1
```
Comment (shutej) [inline schedule/requirements.txt]: so i think you could simply use a git link to the source as we do elsewhere.  that would install dateutil from the correct location without renaming things

Comment (aleontiev) [inline schedule/requirements.txt]: I'll try that; this project uses the old setuptools in the root directory and does not have a requirements.txt file there.. not sure how this file is actually used

File: schedule/models/rules.py [hunk 2]
```diff
@@ -12,76 +12,94 @@
...
-                param = (str(param[0]), [int(p) for p in param[1].split(',')])
-                if len(param[1]) == 1:
-                    param = (param[0], param[1][0])
-                param_dict.append(param)
-        return dict(param_dict)
-
-    def __unicode__(self):
-        """Human readable string for Rule"""
-        return 'Rule %s params %s' % (self.name, self.params)
+    if self.params is None:
```
Comment (sganeriwal) [inline schedule/models/rules.py:87]: Freaking github is horrible. There are minimal changes to the file and it is showing all types of false negatives.

Comment (altschool-jira) [inline schedule/models/rules.py:87]: This looks pretty good to me. My +1; wait for Jeremy's +1 perhaps.

Comment (aleontiev) [inline schedule/models/rules.py:87]: Sorry, I autoF'd the file =/

File: tests/test_event.py [hunk 3]
```diff
@@ -69,6 +70,31 @@ def test_recurring_event_get_occurrences(self):
...
+        recurring_event = self.__create_recurring_event(
+          'Recurring event test get_occurrences over DST boundary',
+          before_dst.astimezone(utc),
+          before_dst.astimezone(utc),
+          after_dst.astimezone(utc),
+          rule,
+          calendar)
+
+        occurrences = recurring_event.get_occurrences(before_dst, after_dst)
+        self.assertEquals([before_dst, after_dst], [o.start for o in occurrences])
```
Comment (shutej) [inline tests/test_event.py:96]: let's do "hey this is 7 hours after utc in one case and 8 after the other"

**Pull request merged**